streams/src/test/java/org/apache/kafka/streams/integration/CleanUpIntegrationTest.java

```diff
@@ -169,34 +170,34 @@ private KStreamBuilder setupTopology() {
-                                                        public KeyValue<Long, String> apply(final Long key, final String value) {
-                                                            return new KeyValue<>(key, value);
-                                                        }
-                                                    })
-                                                    .groupByKey()
-                                                    .count("global-count");
+            .map(new KeyValueMapper<Long, String, KeyValue<Long, String>>() {
+                @Override
+                public KeyValue<Long, String> apply(final Long key, final String value) {
+                    return new KeyValue<>(key, value);
+                }
+            })
+            .groupByKey()
+            .count("global-count");
         globalCounts.to(Serdes.Long(), Serdes.Long(), OUTPUT_TOPIC);
 
         final KStream<Long, Long> windowedCounts = input
-                                                       .through(INTERMEDIATE_USER_TOPIC)
-                                                       .map(new KeyValueMapper<Long, String, KeyValue<Long, String>>() {
-                                                           @Override
-                                                           public KeyValue<Long, String> apply(final Long key, final String value) {
-                                                               Utils.sleep(1000);
-                                                               return new KeyValue<>(key, value);
-                                                           }
-                                                       })
-                                                       .groupByKey()
-                                                       .count(TimeWindows.of(35).advanceBy(10), "count")
-                                                       .toStream()
-                                                       .map(new KeyValueMapper<Windowed<Long>, Long, KeyValue<Long, Long>>() {
-                                                           @Override
-                                                           public KeyValue<Long, Long> apply(final Windowed<Long> key, final Long value) {
-                                                               return new KeyValue<>(key.window().start() + key.window().end(), value);
-                                                           }
-                                                       });
+            .through(INTERMEDIATE_USER_TOPIC)
+            .map(new KeyValueMapper<Long, String, KeyValue<Long, String>>() {
+                @Override
+                public KeyValue<Long, String> apply(final Long key, final String value) {
+                    Utils.sleep(1000);
+                    return new KeyValue<>(key, value);
+                }
+            })
+            .groupByKey()
+            .count(TimeWindows.of(35).advanceBy(10), "count")
```

Comment:
Window parameters are not important. Of course, they should somehow align with input data timestamps -- I want to make sure to get many windows and not just one.

For the sleep time: it must only ensure, that we get at least one value processed (but this is not a problem, as we wait for a result value to arrive) and must avoid processing all data (if all data is processed, test would not fail, but would be void as we want to "skip over" unprocessed data in clean-up) -- thus, a smaller value might void the test while a larger value only increases test run time.